src/main/cpp/commands/swerve/AutoSwerveCommand.cpp

```diff
@@ -0,0 +1,49 @@
+
+#include "commands/swerve/AutoSwerveCommand.h"
+
+AutoSwerveCommand::AutoSwerveCommand(std::shared_ptr<SwerveDrive> swerve, frc::Pose2d f_pos, bool keep_init_heading) {
+    swerve_ = swerve;
+    end_pos = f_pos;
+    keep_heading = keep_init_heading;
+
+    if (swerve_ != nullptr) {
+        this->AddRequirements(swerve_.get());
+    }
+}
+
+void AutoSwerveCommand::Initialize() {
+    VOKC_CHECK(swerve_ != nullptr);
+
+    VOKC_CALL(swerve_->ResetDriveEncoders());
+    VOKC_CALL(swerve_->ResetPIDs());
+
+    VOKC_CALL(swerve_->InitAuto(end_pos, keep_heading));
+}
```

Comment:
So is this command supposed to run the whole autonomous? I was assuming we'd do something like send poses and return true in isFinished always, and let the subsystem run the show. Is this a placeholder command?

I'm fine with this as a placeholder as long as we work on this logic for #16